### ISSUE TITLE
Highlighting problems with fields called "type"

### ISSUE THREAD

**Issue by alaiacano** [spotify/scio-idea-plugin#5]
If my query return an object with a `.type` value, the IDE does not recognize the field when accessing it via ``` a.`type` ```. It's highlighted in red, though the job _does_ compile properly.

```scala
@BigQueryType.fromQuery(
  """
    |select
    |  some_object.type
    |from [my-table]
    |limit 10
  """.stripMargin) class TypeField

  sc.typedBigQuery[TypeField]()
    .map { a => a.some_object.`type` }   // `type` is highlighted in red, doesn't show up on auto-complete, etc
```

**Comment by ravwojdyla** [comment]
This problem requires change in scio itself, will be fixed in release with spotify/scio#523.

**Comment by ravwojdyla** [comment]
Solved - tested on IDEA:

```
IntelliJ IDEA 2016.3.4
Build #IC-163.12024.16, built on January 31, 2017
JRE: 1.8.0_112-release-408-b6 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

Scio plugin version: `0.1.5`
Scio version: `0.3.1`
